release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -4114,4 +4114,23 @@ $ oc adm release info 4.10.64 --pullspecs
 [id="ocp-4-10-64-upgrading"]
 ==== Updating
 
-To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
\ No newline at end of file
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-10-65"]
+=== RHBA-2023:4445 - {product-title} 4.10.65 bug fix update
+
+Issued: 2023-08-09
+
+{product-title} release 4.10.65 is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2023:4445[RHBA-2023:4445] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:4447[RHBA-2023:4447] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.65 --pullspecs
+----
+
+[id="ocp-4-10-65-upgrading"]
+==== Updating
+
+To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
```

Comment:
In the preview, I see that the link for the "Updating a cluster using the CLI" section is giving the 404 error page. All the same links in other sections are also giving the 404 error page. Do you know why this could be happening? Can you please fix that?

Reply:
Thanks for the reply and for catching the link that does not work because of my manual preview link. I can confirm that the links do work as per the published doc: https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-64

I copied the `==== Updating` section from 4.10.64 and applied this exact section to 4.10.65.

Reply:
+1 @xenolinux 
@dfitzmau Thanks for confirming that the links work per published documentation. However, I peer-reviewer/merge-reviewer will still still need to verify that it does. May I suggest uploading documentation for preview using the manual method. (You can find the details in the Docs Manual). I have done so using the same to verify the link:
https://file.rdu.redhat.com/avbhatt/pr-63283/release_notes/ocp-4-10-release-notes.html#ocp-4-10-65